File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/superclasses/Conveyor.java
package org.firstinspires.ftc.teamcode.superclasses;

public interface Conveyor extends RobotModule{
    void setConveyorPower(double power);
    void setForceReverse(boolean forceReverse);
    void setReverseAfterStop(boolean doReverseOnStop);
    void setAutomaticConveyorStopping(boolean doAutomaticConveyorStopping);
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/superclasses/Conveyor.kt
package org.firstinspires.ftc.teamcode.superclasses;

public interface Conveyor {

    void enableConveyor(boolean isEnabled);

    void setForceReverse(boolean forceReverse);

    void setReverseAfterStop(boolean doReverseOnStop);

    void setAutomaticConveyorStopping(boolean doAutomaticConveyorStopping);
}
